test/gtest/group_conv3d_wrw.cpp

```diff
@@ -35,3 +35,4 @@ DEFINE_GROUP_CONV3D_TEST(float, BackwardWeights);
 // DEFINE_GROUP_CONV3D_TEST(half, BackwardWeights);
 /// \todo int8_t tests don't work. Need debugging
 // DEFINE_GROUP_CONV3D_TEST(int8_t, BackwardWeights);
+DEFINE_GROUP_CONV3D_TEST(bfloat16, BackwardWeights);
```

Comment:
This looks good. Please try uncommenting `half` for backward weights to see if it's still broken.